monopod/pkg/images/images.go

```diff
@@ -1,82 +1,20 @@
 package images
 
 import (
-	"fmt"
 	"os"
-	"path"
-	"path/filepath"
-	"strings"
-
-	"chainguard.dev/apko/pkg/build/types"
-	"gopkg.in/yaml.v3"
 
 	"github.com/chainguard-images/images/monopod/pkg/constants"
 )
 
 type Image struct {
-	ImageName                   string `json:"imageName"`
-	ImageStatus                 string `json:"imageStatus"`
-	ImageSummaryJson            string `json:"imageSummaryJson"`
-	ApkoConfig                  string `json:"apkoConfig"`
-	ApkoKeyringAppend           string `json:"apkoKeyringAppend"`
-	ApkoRepositoryAppend        string `json:"apkoRepositoryAppend"`
-	ApkoAdditionalTags          string `json:"apkoAdditionalTags"`
-	ApkoBaseTag                 string `json:"apkoBaseTag"`
-	ApkoTargetTag               string `json:"apkoTargetTag"`
-	ApkoTargetTagSuffix         string `json:"apkoTargetTagSuffix"`
-	ApkoPackageVersionTag       string `json:"apkoPackageVersionTag"`
-	ApkoPackageVersionTagPrefix string `json:"apkoPackageVersionTagPrefix"`
-	ApkoPackageAppend           string `json:"apkoPackageAppend"`
-	TestCommandExe              string `json:"testCommandExe"`
-	TestCommandDir              string `json:"testCommandDir"`
-	ExcludeTags                 string `json:"excludeTags"`
-	UseTerraform                bool   `json:"useTerraform"`
-	TerraformDirectory          string `json:"terraformDirectory"`
-	ExcludeContact              bool   `json:"-"`
-}
-
-type ImageManifest struct {
-	Ref            string                       `yaml:"ref"`
-	Status         string                       `yaml:"status"`
-	ExcludeContact bool                         `yaml:"excludeContact"`
-	Terraform      *bool                        `yaml:"terraform,omitempty"`
-	Variants       []ImageManifestVariant       `yaml:"versions"`
-	Options        map[string]types.BuildOption `yaml:"options,omitempty"`
-}
-
-type ImageManifestVariant struct {
-	Apko ImageManifestVariantApko `yaml:"apko"`
-}
-
-type ImageManifestVariantApko struct {
-	Config          string                                  `yaml:"config"`
-	Options         []string                                `yaml:"options"`
-	ExtractTagsFrom ImageManifestVariantApkoExtractTagsFrom `yaml:"extractTagsFrom"`
-	Tags            []string                                `yaml:"tags"`
-	Subvariants     []ImageManifestVariantApkoSubvariant    `yaml:"subvariants"`
-}
-
-type ImageManifestVariantApkoSubvariant struct {
-	Suffix  string   `yaml:"suffix"`
-	Options []string `yaml:"options"`
-}
-
-type ImageManifestVariantApkoExtractTagsFrom struct {
-	Package string   `yaml:"package"`
-	Prefix  string   `yaml:"prefix"`
-	Exclude []string `yaml:"exclude"`
+	ImageName string `json:"imageName"`
```

Comment:
If there's nothing else in here we want to keep, this can just be a `string`. We usually consume it as a `[]Image`, which could be a `[]string`.